src/tests/system/mhc.yaml

```diff
@@ -31,6 +31,10 @@ domains:
         ipa_domain: ipa.test
         krb5_keytab: /var/enrollment/ipa.test.keytab
         ldap_krb5_keytab: /var/enrollment/ipa.test.keytab
+    artifacts:
+    - /etc/sssd/*
+    - /var/log/*
```

Comment:
This includes KDC and DS?
Won't be too heavy? On my laptop it's currently 1.7G :-/